tests/integration/PwmService.test.ts

```diff
@@ -0,0 +1,72 @@
+import { hasUncaughtExceptionCaptureCallback } from "process"
+import { PWMService } from "../../src"
+import { PWMPins, Polarity } from "../../src"
+
+jest.unmock('zeromq')
+
+describe('PwmService', () => {
+    let pwm: PWMService
+
+    beforeAll(async () => {
+        pwm = new PWMService('tcp://localhost:5555')
+        await pwm.initPwm(PWMPins.PWM1)
+        await pwm.initPwm(PWMPins.PWM2)
+    })
+
+    const enumNames: { [key: number]: string } = {
+        0: 'PWMPins.PWM1',
+        1: 'PWMPins.PWM2'
+    }
+    
+    test.each([
+        [{pwmNum: PWMPins.PWM1, frequency: 1000, dutyCycle: 1, polarity: Polarity.NORMAL}],
+        [{pwmNum: PWMPins.PWM2, frequency: 1000, dutyCycle: 1, polarity: Polarity.NORMAL}]
+    ])('Call setConfig and check configurations', async (args) => {
+        const response = await pwm.setConfig(args)
+        expect(response).toEqual('Successfully applied pwm configurations.')
+
+        const frequency = await pwm.getFrequency(args.pwmNum)
+        expect(typeof frequency).toBe('number')
```

Comment:
`typeof `is fine to use, but not for general type checking. I would recommend using Javascipt static methods to check:
For example: 
expect(Number.isFinite(dutyCycle)).toBe(true);